lib/src/special_elements.dart

```diff
@@ -99,12 +99,12 @@ class SpecialClasses {
 
   /// Add a class object that could be special.
   void addSpecial(Class aClass) {
-    if (_specialClassDefinitions.containsKey(aClass.name)) {
-      var d = _specialClassDefinitions[aClass.name]!;
-      if (d.matchesClass(aClass)) {
-        assert(!_specialClass.containsKey(d.specialClass) ||
-            _specialClass[d.specialClass] == aClass);
-        _specialClass[d.specialClass] = aClass;
+    var def = _specialClassDefinitions[aClass.name];
```

Comment:
Let's go ahead and call this 'definition'.

https://dart.dev/guides/language/effective-dart/documentation#avoid-abbreviations-and-acronyms-unless-they-are-obvious